codegen.yml

```diff
@@ -2,8 +2,7 @@ overwrite: true
 schema: 'http://localhost:8000/___graphql'
 documents:
   - cache/fragments/**/*.js
-  - node_modules/gatsby*/!(node_modules)/**/*.js
-  - node_modules/gatsby-*/**/*.js
+  - node_modules/gatsby-transformer-sharp/!(node_modules)/**/*.js
```

Comment:
Had to reduce glob pattern here because graphql typegen was broken.

Reply:
The generation was crashing because the glob was too large. This is actually an issue related to the gatsby version we're using.